lte/gateway/python/magma/pipelined/app/access_control.py

```diff
@@ -154,29 +159,56 @@ def _add_gre_tun_allow_flow(self, datapath, gre_ip, gre_key):
             resubmit_table=self.next_table,
         )
 
-    def _install_local_ip_blocking_flows(self, datapath):
+    def _install_local_ip_blocking_flows(self, datapath, ip_version):
```

Comment:
Since we call this method for both versions and it's not used anywhere else, it might be easier to just block both ip versions in this method and not have the parameter.